integration-tests/gradle/src/test/java/io/quarkus/gradle/ListExtensionsTest.java

```diff
@@ -0,0 +1,41 @@
+package io.quarkus.gradle;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ListExtensionsTest extends QuarkusPlatformAwareGradleTestBase {
+
+    @Test
+    public void testListExtensionsWork() throws IOException, URISyntaxException, InterruptedException {
+
+        final File projectDir = getProjectDir("list-extensions-single-module");
+        runGradleWrapper(projectDir, ":listExtension");
+
+        List<String> outputLogLines = listExtensions(projectDir,":listExtension");
+
+        assertThat(outputLogLines).anyMatch(line -> line.contains("quarkus-resteasy"));
+        assertThat(outputLogLines).anyMatch(line -> line.contains("quarkus-vertx"));
+    }
+
+
+
+    private List<String> listExtensions(File projectDir, String... args) throws IOException, InterruptedException {
+//        List<String> command = new LinkedList<>();
+//        command.add(getPluginGroupId() + ":" + getPluginArtifactId() + ":" + getPluginVersion() + ":listExtension");
```

Comment:
Let's remove these comments please

Reply:
Would these be required ?
While working on this test with Rohan sir, he stated that these would be needed, and thus we made ```QuarkusPlatformAwareGradleTestBase``` , but with what I've done yday, this isn't being used, so should I remove this class too ?